rtgui/filmnegative.cc

```diff
@@ -565,7 +542,7 @@ bool FilmNegative::mouseOver(int modifierKey)
 {
     EditDataProvider* const provider = getEditProvider();
     Rectangle* const spotRect = static_cast<Rectangle*>(visibleGeometry.at(0));
-    spotRect->setXYWH(provider->posImage.x - 16, provider->posImage.y - 16, 32, 32);
+    spotRect->setXYWH(provider->posImage.x - *displayRectWidth, provider->posImage.y - *displayRectWidth, *displayRectWidth * 2, *displayRectWidth * 2);
```

Comment:
It is half the length of the rectangle. This is what the user  selects as the size of the spot. It could be changed to the full length, but then the size wouldn't match the corresponding size of the WB picker. I'm renaming the variable to `_spotSideLength` but `_spotSideHalfLength` might be more appropriate. What do you think?

Reply:
Apparently, it's called an "apothem". I don't think many people know that word, so size or half width is better.